project/frontend/src/App.js

```diff
@@ -1,26 +1,29 @@
 import './App.css'
 
 import React from 'react';
-import ReactDOM from 'react-dom';
-import {BrowserRouter, Route} from 'react-router-dom'
+import {Route} from 'react-router-dom'
 
 import CreateCompetition from './containers/CreateCompetition/CreateCompetition'
 import Explanation from './containers/Explanation/Explanation'
 import Login from './containers/Login/Login'
 import SelectCompetition from './containers/SelectCompetition/SelectCompetition'
 import SGonksPlatfrom from './containers/SGonksPlatform/SGonksPlatform'
 
+import HeaderBar from "./components/HeaderBar/HeaderBar";
+
 function App() {
+
```

Comment:
Is there a reason for all this blank space?

Reply:
This has been fixed in the next PR :)